File: Problems/Minimum Number of Chairs in a Waiting Room.py
class Solution:
    def minimumChairs(self, s: str) -> int:
        ma = 0
        curr = 0
        for i in s:
            if i == 'E':
                curr += 1
                ma = max(ma, curr)
            else:
                curr -= 1
        return ma

File: Problems/Minimum Number of Chairs in a Waiting Room.cpp
class Solution {
public:
    int minimumChairs(string s) {
        int min = 0, temp = 0;
        for(int i = 0, n = s.length(); i < n; ++i){
            temp += (s[i] == 'E') ? 1 : -1;
            if(temp > min) min = temp;
        }
        return min;
    }
};
